### PR TITLE
add voting system and endgame

### PR DESCRIPTION
**Pull request by SylvanSign**
Not the best, but works I think

### CODE REVIEW

**Comment by SylvanSign** [comment]
might be easier to review together in person

File: randomWords.js [hunk 1]
```diff
@@ -3,9 +3,17 @@
...
 function getRandomWords(wordCount) {
+  let words
+  do {
+    words = randomWords(wordCount)
+  } while (!allElemsUnique(words))
   return randomWords(wordCount)
 }
 
+function allElemsUnique(arr) {
+  return arr.length === new Set(arr).size
```
Comment (SylvanSign) [inline randomWords.js:14]: I had a case where "sail" appeared twice for me in my required words list. Idk if that's part of the contract for the library or what, but this was easy fix for now

File: server.js [hunk 2]
```diff
@@ -16,8 +16,8 @@ const {
 } = require('./rooms')
 const roundManager = require('./roundManager')
 
-const MIN_PLAYER_COUNT = 3
-const START_GAME_DELAY = 5 // sec
+const MIN_PLAYER_COUNT = 1
```
Comment (ZakMiller) [inline server.js]: woops

File: roundManager.js [hunk 3]
```diff
@@ -2,9 +2,10 @@
 
 let io
 
-const NUM_ROUNDS = 5
+const NUM_ROUNDS = 2
```
Comment (ZakMiller) [inline roundManager.js]: woops

Comment (SylvanSign) [reply roundManager.js]: I'll fix this in a bit. A couple other things I want to add too

**Review by ZakMiller — APPROVED** [review]

**Pull request merged**